contracts/TimeManagerV5.sol

```diff
@@ -0,0 +1,61 @@
+// SPDX-License-Identifier: BSD-3-Clause
+pragma solidity 0.5.16;
+
+contract TimeManagerV5 {
+    /// @dev The approximate number of seconds per year
+    uint256 public constant SECONDS_PER_YEAR = 31_536_000;
+
+    /// @notice Number of blocks per year or seconds per year
+    uint256 public blocksOrSecondsPerYear;
+
+    /// @dev Sets true when block timestamp is used
+    bool public isTimeBased;
+
```

Comment:
Should we add a gap, given this contract will be extended?

Reply:
[e3f9df8](https://github.com/VenusProtocol/solidity-utilities/pull/8/commits/e3f9df8370b72d3f0631b2547911a96d4451385c)